index.html

```diff
@@ -33,7 +33,7 @@
 				coreEl.setAttribute('src', '/packages/core/lib/core.umd.js');
 				document.body.appendChild(coreEl);
 				coreEl.onload = function () {
-					var components = ['button'];
+					var components = ['icon', 'typography', 'button', 'overlay', 'select-menu', 'list-item', 'select'];
```

Comment:
this changes are not needed, but maybe they can remain here as in another example of using another more complex component

Reply:
I'm working on an improvement for this, hopefully each component would have their own development file with the proper setup to make the process easier.

Reply:
would be nice. let me know if you want to keep this changes or should I revert it